tests/Tests.Benchmarking/BulkBenchmarkLowLevelTests.cs

```diff
@@ -0,0 +1,155 @@
+
+		[Benchmark(Description = "StaticListOfObjects")]
+		public BulkResponse StaticListOfObjects()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.MultiJson(StaticArrayOfObjects));
+			return lowLevel;
+		}
+
+		private static readonly PostData StaticPostData = PostData.MultiJson(StaticArrayOfObjects);
+		[Benchmark(Description = "PrecomputedPostDataOfObjects")]
+		public BulkResponse PrecomputedPostDataOfObjects()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(StaticPostData);
+			return lowLevel;
+		}
+
+		private static readonly string _header = @"{""index"":{}}";
+		private static readonly string[] StaticListString =
+			Projects.SelectMany(p => new string[] { _header, Client.SourceSerializer.SerializeToString(p) }).ToArray();
+		private static readonly PostData StaticPostDataListString = PostData.MultiJson(StaticListString);
+
+		[Benchmark(Description = "StaticListOfStrings")]
+		public BulkResponse StaticListOfStrings()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.MultiJson(StaticListString));
+			return lowLevel;
+		}
+
+		[Benchmark(Description = "PrecomputedPostDataOfStrings")]
+		public BulkResponse PrecomputedPostDataOfStrings()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(StaticPostDataListString);
+			return lowLevel;
+		}
+
+		private static readonly string StaticString = string.Join("\n", StaticListString);
+		[Benchmark(Description = "PostDataString")]
+		public BulkResponse PostDataString()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.String(StaticString));
+			return lowLevel;
+		}
+
+		private static readonly PostData PrecomputedStaticString = PostData.String(StaticString);
+		[Benchmark(Description = "PrecomputedPostDataString")]
+		public BulkResponse PrecomputedPostDataString()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PrecomputedStaticString);
+			return lowLevel;
+		}
+
+		private static readonly byte[] StaticBytes = Encoding.UTF8.GetBytes(StaticString);
+		[Benchmark(Description = "PostDataByteArray")]
+		public BulkResponse PostDataByteArray()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.Bytes(StaticBytes));
+			return lowLevel;
+		}
+
+		private static readonly ReadOnlyMemory<byte> StaticMemory = StaticBytes.AsMemory();
+
+		[Benchmark(Description = "PostDataReadOnlyMemory")]
+		public BulkResponse PostDataReadOnlyMemory()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.ReadOnlyMemory(StaticMemory));
+			return lowLevel;
+		}
+
+		[Benchmark(Description = "PostDataStreamHandler")]
+		public BulkResponse PostDataStreamHandler()
+		{
+			var lowLevel = ClientLowLevel.Bulk<BulkResponse>(PostData.StreamHandler(StaticBytes,
+				(bytes, s) => s.Write(bytes.AsSpan()),
+				async (bytes, s, ctx) => await s.WriteAsync(bytes.AsMemory(), ctx)));
+			return lowLevel;
+		}
+
+		[GlobalCleanup]
+		private static void X()
```

Comment:
Might be worth commenting why this is needed?

Reply:
Its not will remove 👍